device-registry/src/main/scala/org/genivi/sota/device_registry/db/GroupInfoRepository.scala

```diff
@@ -28,10 +29,11 @@ object GroupInfoRepository extends SlickJsonHelper {
 
   val groupInfos = TableQuery[GroupInfoTable]
 
-  def list(namespace: Namespace)(implicit ec: ExecutionContext): DBIO[Seq[Group]] =
+  def list(namespace: Namespace, offset:Option[Long], limit: Option[Long])
+          (implicit ec: ExecutionContext): DBIO[PaginatedResult[Group]] =
     groupInfos
       .filter(g => g.namespace === namespace)
-      .result
+      .paginatedResult(offset, limit)
```

Comment:
I don't know who wrote this whole paginated thing but this is a quite nice API, both the slick/db side as well as the http side. 👍 👍